batch-1/movie-info/movie_details.rb

```diff
@@ -0,0 +1,29 @@
+ require 'json'
+ require 'open-uri'
+puts"Hello,Enter the movie name:\n"
+usr_input=gets.chomp
+usr_input=usr_input.split(" ")# Check whether the entered movie is more than one word
+usr_input=usr_input.join("+")#If yes,add + in between the words.
+movie_url= "http://www.omdbapi.com/?t=#{usr_input}&y=&plot=short&r=json"
+content = open(movie_url).read
+content_hash=JSON.parse(content)
+html_file=File.new("movie_detail.html","w")
+new_line = "\n"
+html_file.puts("<!DOCTYPE html>"+new_line)
```

Comment:
Instead of writing HTML inside ruby code, how about this approach:

2 files: movie_details.rb and template.html

In ```template.haml```, we would write something like this:

```
....
<body>
  <h1>{{title}}</h1>
</body>
...
```

We would open ```template.html``` in ruby code and replace ```{{title}}``` with appropriate text using string replace methods. That way we could take the entire html out of ruby code and keep it in a separate template file. Give that a try.